homeassistant/components/pid_thermostat/config_flow.py

```diff
@@ -0,0 +1,103 @@
+"""Config flow for pid integration."""
+from collections.abc import Mapping
+import logging
+from typing import Any, cast
+
+import voluptuous as vol
+
+from homeassistant.components.number import DOMAIN as NUMBER_DOMAIN
+from homeassistant.components.pid_controller.config_flow import (
```

Comment:
Integrations are not allowed to consume others.

Import from another integration is thus not allowed.